ivy_tests/test_ivy/test_frontends/test_torchvision/confest.py

```diff
@@ -0,0 +1,6 @@
+import pytest
```

Comment:
The file should be named `conftest` not `confest`. That's why you still can't run the tests 😅

Reply:
😆 😆 🤣